tests/Printer/XmlTest.php

```diff
@@ -66,7 +69,9 @@ public function testPrintData() : void
     <files>
         <file errors="1" path="Foo/Bar.php">
             <entry end="82" line="1" start="80">
-                <snippet><![CDATA[$rootNode->setAttribute('fileCount', count($fileReportList->getFileReports()) + 12);]]></snippet>
+                <snippet>
+                    <![CDATA[$rootNode->setAttribute('fileCount', count($fileReportList->getFileReports()) + 12);]]>
```

Comment:
This 12 was removed wasn't it ?

Reply:
as you can see :) I just fixed XML to follow our standards.